backend/apps/content/tests_rbac.py

```
@@ -0,0 +1,99 @@
+        self.client.force_authenticate(user=self.regular_user)
+        response = self.client.get("/api/content/learning-paths/")
+        assert response.status_code == 200
+        results = (
+            response.data
+            if isinstance(response.data, list)
+            else response.data.get("results", [])
+        )
+        path_access_map = {
+            p["id"]: bool(p.get("hasAccess", p.get("has_access"))) for p in results
+        }
+        assert path_access_map[self.public_path.id] == True
```

Ruff (0.16.0)

[error] Avoid equality comparisons to `True`; use `path_access_map[self.public_path.id]:` for truth checks

Replace with `path_access_map[self.public_path.id]`

(E712)